src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/Sentence.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppstier.domain
+
+import uk.gov.justice.digital.hmpps.hmppstier.client.SentenceDto
+import java.time.LocalDate
+
+data class Sentence constructor(
+  val terminationDate: LocalDate?,
+  val sentenceType: String,
+  val startDate: LocalDate,
+  val expectedSentenceEndDate: LocalDate?,
```

Comment:
it's very minor but we don't need to repeat `sentence` here just because it's the field name in community-api.